packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+	<header>
+		<div class="title-row">
+			<div class="title-container">
+				<h4><slot /></h4>
+				<i v-if="props.tooltip" v-tooltip="tooltip" class="pi pi-info-circle" />
+			</div>
+			<Button
+				class="close-mask"
+				icon="pi pi-times"
+				text
+				rounded
+				aria-label="Close"
+				@click="emit('close')"
+			/>
+		</div>
+		<div class="tabs-row">
+			<div>
```

Comment:
why the extra `<div>`?

Reply:
This was a way so that if there were no tabs, there would an empty element so that the Documentation link would always stick to the right:

```
header > * {
	display: flex;
	align-items: center;
	justify-content: space-between;
}
```

Reply:
That why using CSS grid would avoid those kinds of hacks. Though you could do this just with flex-box.

Reply:
div has been removed, Ive kept flexbox. Did it with `margin-left: auto` on the <a> tag instead